.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,15 @@
+name: Feature request
+description: Propose a new feature or enhancement for PDF.js
+title: "[Feature]: "
+body:
```

Comment:
Given that the Firefox PDF Viewer is the primary development target, how about adding a *required* Yes/No question at the top to help highlight that?
`Is the feature relevant to the Firefox PDF Viewer?`